diff --git a/README.md b/README.md
index b964c153..ad6eed7d 100644
--- a/README.md
+++ b/README.md
@@ -1,3 +1,17 @@
+# Welcome
+This project is the [bananaCatCo Store] An Isomorphic (first render on the server, then Single-page App on the client) React Application for E-Commerce Storefronts.
+
+# To do:
+-el stock number no se actualiza despues de la compra  --------done
+-cambiar en AdminProducts la luz verde por el numero de stock --------done
+-agregar botones de eliminar --------done
+
+
+-agregar popups cuando queres comprar mas del stock que hay
+-cambiar nombres scss a DeleteHandler
+-cuando eliminas un producto. borrarlo de los carts
+- crear filtros en Pedidos tabs de creados, por enviar, pagados,etc
+
 # Welcome
 ### Isomorphic?
 The [Single-Page Apps (SPA)](https://en.wikipedia.org/wiki/Single-page_application) paradigm has suffered from some bad rep (in certain applications like... e-commerce) mainly because of:
diff --git a/src/api/products.js b/src/api/products.js
index dfed1ed0..0f006fb6 100644
--- a/src/api/products.js
+++ b/src/api/products.js
@@ -55,6 +55,16 @@ class ProductsAPI {
         });
     }
 
+    /**
+     * Delete Product
+     */
+    delete(productId) {
+        return new Promise((resolve, reject) => {
+            let request = superagent.delete(`${this.baseUrl}/products/${productId}`);
+            this._wrapAndRequest(request, resolve, reject);
+        });
+    }
+
     /**
      * Fetch Product with given ID
      */
diff --git a/src/components/common/buttons/DirectionButton.scss b/src/components/common/buttons/DirectionButton.scss
index feec5c1e..e5a1aef9 100644
--- a/src/components/common/buttons/DirectionButton.scss
+++ b/src/components/common/buttons/DirectionButton.scss
@@ -9,6 +9,12 @@
     &:not(:first-child) {
         margin-left: 20px;
     }
+    @media only screen and (max-width: 400px) {
+        &:not(:first-child) {
+            margin-left: 0px;
+        }
+        float: none;
+    }
 }
 
 .direction-button__move-button {
diff --git a/src/components/common/images/LogoDisplayer.scss b/src/components/common/images/LogoDisplayer.scss
index 82cfd1b1..0f453aea 100644
--- a/src/components/common/images/LogoDisplayer.scss
+++ b/src/components/common/images/LogoDisplayer.scss
@@ -2,7 +2,7 @@
   // img {
   //     width: 58px;
   //     height: 52px;
-  //     background: url('../../../../../static/logo.png');
+  //     // background: url('../../../../../static/logo.png');
   //     background-size: 100%;
   //     background-repeat: no-repeat;
   // }
@@ -13,4 +13,11 @@
 
 .desktop-header__logo-link {
   align-self: center;
+
+  img {
+    width: 58px;
+    height: 58px;
+    background-size: 100%;
+    background-repeat: no-repeat;
+}
 }
\ No newline at end of file
diff --git a/src/components/common/navigation/MainNavigation.scss b/src/components/common/navigation/MainNavigation.scss
index 3cea755b..eb5e47b2 100644
--- a/src/components/common/navigation/MainNavigation.scss
+++ b/src/components/common/navigation/MainNavigation.scss
@@ -3,6 +3,16 @@
 .main-navigation {
     width: 100%;
     font-family: $font-family-heading;
+
+    nav {
+        ul {
+            @media only screen and (max-width: 400px) {
+                flex-direction: column;
+                display: flex;
+                text-align: left;
+            }
+        }
+    }
 }
 
 .main-navigation__item {
diff --git a/src/components/common/navigation/SideMenu.js b/src/components/common/navigation/SideMenu.js
index 6893cef7..7967458d 100644
--- a/src/components/common/navigation/SideMenu.js
+++ b/src/components/common/navigation/SideMenu.js
@@ -10,6 +10,7 @@ import PropTypes from 'prop-types';
 // Flux
 import AccountStore from '../../../stores/Account/AccountStore';
 import triggerDrawer from '../../../actions/Application/triggerDrawer';
+import IntlStore from '../../../stores/Application/IntlStore';
 
 // Required Components
 import Text from '../typography/Text';
@@ -52,6 +53,9 @@ class SideMenu extends React.Component {
     //*** Template ***//
 
     render() {
+        const locale = this.context.getStore(IntlStore).getCurrentLocale();
+        const routeParams = {locale: locale};
+        const isAdmin = this.context.getStore(AccountStore).isAuthorized(['admin']);
         return (
             <div className="side-menu">
                 <nav>
@@ -91,6 +95,19 @@ class SideMenu extends React.Component {
                                     </div>
                                 </Link>
                             </li>
+                            <li className="side-menu__item side-menu__account-item" onClick={this.handleItemClick}>
+                                { isAdmin ?
+                                    <div className="desktop-header__logout-button">
+                                        <Link to={`/${this.context.intl.locale}/adm`} params={routeParams}>
+                                            <Text size="small" weight="bold" color="white">
+                                                <FormattedMessage id="adminPanel" />
+                                            </Text>
+                                        </Link>
+                                    </div>
+                                    :
+                                    null
+                                }
+                            </li>
                             <li className="side-menu__item side-menu__account-item" onClick={this.handleItemClick}>
                                 <Link to={`/${this.context.intl.locale}/logout`} >
                                     <Text size="small" weight="bold">
diff --git a/src/components/common/tables/Table.scss b/src/components/common/tables/Table.scss
index e98ba29c..93d9d1fc 100644
--- a/src/components/common/tables/Table.scss
+++ b/src/components/common/tables/Table.scss
@@ -4,12 +4,23 @@
     width: 100%;
     border: 1px solid #eeeeee;
     text-align: left;
+    @media only screen and (max-width: 400px) {
+        thead {
+            display: none;
+        }
+    }
 }
 
 .table__row {
     height: 50px;
     border-bottom: 1px solid #eeeeee;
 
+    @media only screen and (max-width: 400px) {
+        height: 100%;
+        flex-direction: column;
+        display: flex;
+    }
+
     &--info {
         background-color: $background-color-table-info;
     }
@@ -31,6 +42,9 @@
 .table__data {
     padding: 0 15px 0 15px;
     vertical-align: middle;
+    @media only screen and (max-width: 400px) {
+        padding: 5px;
+    }
 }
 
 .table__body > .table__row {
diff --git a/src/components/pages/Admin/Admin.scss b/src/components/pages/Admin/Admin.scss
index c965b6a4..c6b4a9dc 100644
--- a/src/components/pages/Admin/Admin.scss
+++ b/src/components/pages/Admin/Admin.scss
@@ -16,10 +16,16 @@
     padding: 20px 30px 20px 30px;
     border-bottom: 3px solid $color-primary;
     background-color: $color-secondary;
+    @media only screen and (max-width: 400px) {
+        flex-direction: column;
+    }
 }
 
 .admin-title {
     align-self: center;
+    @media only screen and (max-width: 400px) {
+        display: none;
+    }
 }
 
 .admin-nav {
diff --git a/src/components/pages/Admin/Collections/AdminCollections.js b/src/components/pages/Admin/Collections/AdminCollections.js
index 4eb35e9d..b3744ea3 100644
--- a/src/components/pages/Admin/Collections/AdminCollections.js
+++ b/src/components/pages/Admin/Collections/AdminCollections.js
@@ -110,11 +110,21 @@ class AdminCollections extends React.Component {
             return {
                 data: [
                     <span className="admin-collections__link">
+                        <div className="admin-collections__mobile-title">
+                            <Text size="medium" weight="bold">
+                                <FormattedMessage id="collectionNameHeading" />
+                            </Text>
+                        </div>
                         <Link to={`/${locale}/adm/collections/${collection.id}`}>
                             {collection.name[locale]}
                         </Link>
                     </span>,
                     <Text size="medium">
+                        <div className="admin-collections__mobile-title">
+                            <Text size="medium" weight="bold">
+                                <FormattedMessage id="collectionParentHeading" />
+                            </Text>
+                        </div>
                         {collection.parentId ?
                             <span>
                                 {this.context.getStore(CollectionsStore).getCollection(collection.parentId).name[locale]}
@@ -124,6 +134,11 @@ class AdminCollections extends React.Component {
                         }
                     </Text>,
                     <Text size="medium">
+                        <div className="admin-collections__mobile-title">
+                            <Text size="medium" weight="bold">
+                                <FormattedMessage id="tagsHeading" />
+                            </Text>
+                        </div>
                         <div className="admin-collections__labels">
                             {collection.tags.map(function (section, idx) {
                                 return (
@@ -136,7 +151,14 @@ class AdminCollections extends React.Component {
                             })}
                         </div>
                     </Text>,
-                    <StatusIndicator status={(collection.enabled === true) ? 'success' : 'default'} />
+                    <div className="admin-collections__status">
+                        <div className="admin-collections__mobile-title">
+                            <Text size="medium" weight="bold">
+                                <FormattedMessage id="enabledHeading" />
+                            </Text>
+                        </div>
+                        <StatusIndicator status={(collection.enabled === true) ? 'success' : 'default'} />
+                    </div>
                 ]
             };
         });
diff --git a/src/components/pages/Admin/Collections/AdminCollections.scss b/src/components/pages/Admin/Collections/AdminCollections.scss
index 94842bb0..af45db71 100644
--- a/src/components/pages/Admin/Collections/AdminCollections.scss
+++ b/src/components/pages/Admin/Collections/AdminCollections.scss
@@ -22,6 +22,10 @@
 
 .admin-collections__add-button {
     width: 130px;
+
+    @media only screen and (max-width: 400px) {
+        width: 90px;
+    }
 }
 
 /*
@@ -33,22 +37,49 @@
 }
 
 .admin-collections__link {
+    @media only screen and (max-width: 400px) {
+        display: flex;
+        flex-direction: row;
+    }
     & > a {
         color: $color-primary;
         text-decoration: none;
-
         &:hover {
             text-decoration: underline;
         }
     }
 }
 
+.admin-collections__status {
+    @media only screen and (max-width: 400px) {
+        display: flex;
+        flex-direction: row;
+    }
+}
+
 .admin-collections__labels {
     display: flex;
+    @media only screen and (max-width: 400px) {
+        flex-direction: column;
+    }
+}
+
+.admin-collections__mobile-title {
+    display: none;
+    @media only screen and (max-width: 400px) {
+        display: flex;
+        padding-right: 10px;
+    }
 }
 
 .admin-collections__label {
     &:not(:first-child) {
         margin-left: 5px;
     }
+    @media only screen and (max-width: 400px) {
+        &:not(:first-child) {
+            margin-left: 0px;
+        }
+        padding: 5px;
+    }
 }
diff --git a/src/components/pages/Admin/Contents/AdminContents.js b/src/components/pages/Admin/Contents/AdminContents.js
index a2912868..3fc0b996 100644
--- a/src/components/pages/Admin/Contents/AdminContents.js
+++ b/src/components/pages/Admin/Contents/AdminContents.js
@@ -122,23 +122,40 @@ class AdminContents extends React.Component {
         let rows = this.state.contents.map(function (content) {
             return {
                 data: [
-                    <Text size="medium">
-                        {content.type ?
-                            <Label>
-                                <FormattedMessage id={content.type} />
-                            </Label>
-                            :
-                            <Label type="error">
-                                <FormattedMessage id="noType" />
-                            </Label>
-                        }
-                    </Text>,
+                    <div>
+                        <div className="admin-contents__mobile-item-container">
+                            <Text size="medium" weight="bold">
+                                <FormattedMessage id="typeHeading" />
+                            </Text>
+                        </div>
+                        <Text size="medium">
+                            {content.type ?
+                                <Label>
+                                    <FormattedMessage id={content.type} />
+                                </Label>
+                                :
+                                <Label type="error">
+                                    <FormattedMessage id="noType" />
+                                </Label>
+                            }
+                        </Text>
+                    </div>,
                     <span className="admin-contents__link">
+                        <div className="admin-contents__mobile-item-container">
+                            <Text size="medium" weight="bold">
+                                <FormattedMessage id="nameHeading" />
+                            </Text>
+                        </div>
                         <Link to={`/${locale}/adm/contents/${content.id}`} >
                             {content.name[locale]}
                         </Link>
                     </span>,
                     <div className="admin-contents__labels">
+                        <div className="admin-contents__mobile-item-container">
+                            <Text size="medium" weight="bold">
+                                <FormattedMessage id="tagsHeading" />
+                            </Text>
+                        </div>
                         {content.tags.map(function (tag, idx) {
                             return (
                                 <div key={idx} className="admin-contents__tag">
@@ -149,7 +166,14 @@ class AdminContents extends React.Component {
                             );
                         })}
                     </div>,
-                    <StatusIndicator status={(content.enabled === true) ? 'success' : 'default'} />
+                    <div className="admin-contents__status-container">
+                        <div className="admin-contents__mobile-item-container">
+                            <Text size="medium" weight="bold">
+                                <FormattedMessage id="enabledHeading" />
+                            </Text>
+                        </div>
+                        <StatusIndicator status={(content.enabled === true) ? 'success' : 'default'} />
+                    </div>,
                 ]
             };
         });
diff --git a/src/components/pages/Admin/Contents/AdminContents.scss b/src/components/pages/Admin/Contents/AdminContents.scss
index 860c1d4c..48484958 100644
--- a/src/components/pages/Admin/Contents/AdminContents.scss
+++ b/src/components/pages/Admin/Contents/AdminContents.scss
@@ -22,6 +22,17 @@
 
 .admin-contents__add-button {
     width: 130px;
+    @media only screen and (max-width: 400px) {
+        width: 90px;
+    }
+}
+
+.admin-contents__mobile-item-container {
+    display: none;
+    @media only screen and (max-width: 400px) {
+        display: flex;
+        padding-right: 10px;
+    }
 }
 
 /*
@@ -42,7 +53,20 @@
     margin-top: 30px;
 }
 
+.admin-contents__status-container {
+    @media only screen and (max-width: 400px) {
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+    }
+}
+
 .admin-contents__link {
+    @media only screen and (max-width: 400px) {
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+    }
     & > a {
         color: $color-primary;
         text-decoration: none;
@@ -55,6 +79,10 @@
 
 .admin-contents__labels {
     display: flex;
+    @media only screen and (max-width: 400px) {
+        display: flex;
+        flex-direction: column;
+    }
 }
 
 .admin-contents__tag {
diff --git a/src/components/pages/Admin/Dashboard/AdminHomepageSettings.scss b/src/components/pages/Admin/Dashboard/AdminHomepageSettings.scss
index 482470df..ce562514 100644
--- a/src/components/pages/Admin/Dashboard/AdminHomepageSettings.scss
+++ b/src/components/pages/Admin/Dashboard/AdminHomepageSettings.scss
@@ -11,6 +11,10 @@
 .admin-homepage-settings__ordering {
     display: flex;
     margin-top: 7.5px;
+
+    @media only screen and (max-width: 400px) {
+        flex-direction: column;
+    }
 }
 
 .admin-homepage-settings__ordering-items {
@@ -20,4 +24,8 @@
 .admin-homepage-settings__ordering-actions {
     align-self: center;
     margin-left: 20px;
+    @media only screen and (max-width: 400px) {
+        margin-left: 0px;
+        margin-top: 15px;
+    }
 }
diff --git a/src/components/pages/Admin/Dashboard/AdminMainNavigation.scss b/src/components/pages/Admin/Dashboard/AdminMainNavigation.scss
index d3b99dba..d4c6926f 100644
--- a/src/components/pages/Admin/Dashboard/AdminMainNavigation.scss
+++ b/src/components/pages/Admin/Dashboard/AdminMainNavigation.scss
@@ -1,5 +1,11 @@
 .admin-main-navigation {
     display: flex;
+
+    @media only screen and (max-width: 400px) {
+        flex-direction: column;
+        display: flex;
+        text-align: left;
+    }
 }
 
 .admin-main-navigation__collections {
@@ -12,4 +18,8 @@
     justify-content: flex-end;
     align-self: center;
     margin-left: 20px;
+    @media only screen and (max-width: 400px) {
+        margin-left: 0px;
+        margin-top: 15px;
+    }
 }
diff --git a/src/components/pages/Admin/Products/AdminProducts.js b/src/components/pages/Admin/Products/AdminProducts.js
index caa68cda..809c1729 100644
--- a/src/components/pages/Admin/Products/AdminProducts.js
+++ b/src/components/pages/Admin/Products/AdminProducts.js
@@ -175,16 +175,56 @@ class AdminProducts extends React.Component {
             return {
                 highlight: (product.enabled === true && product.images.length == 0) ? 'warning' : null,
                 data: [
-                    <Text size="medium">{product.sku}</Text>,
+                    <div className="admin-products__mobile-item-container">
+                        <div className="admin-products__mobile-title">
+                            <Text size="medium" weight="bold">
+                                <FormattedMessage id="skuHeading" />
+                            </Text>
+                        </div>
+                        <Text size="medium">{product.sku}</Text>
+                    </div>,
                     <span className="admin-products__link">
+                        <div className="admin-products__mobile-title">
+                            <Text size="medium" weight="bold">
+                                <FormattedMessage id="nameHeading" />
+                            </Text>
+                        </div>
                         <Link to={`/${locale}/adm/products/${product.id}`} >
                             {product.name[locale]}
                         </Link>
                     </span>,
-                    <StatusIndicator status={(product.stock > 0) ? 'default' : 'error'} />,
-                    <StatusIndicator status={(product.images.length > 0) ? 'default' : 'error'} />,
-                    <Text size="medium">{getProductSections(product)}</Text>,
-                    <StatusIndicator status={(product.enabled === true) ? 'success' : 'default'} />
+                    <div className="admin-products__mobile-item-container">
+                        <div className="admin-products__mobile-title">
+                            <Text size="medium" weight="bold">
+                                <FormattedMessage id="stockHeading" />
+                            </Text>
+                        </div>
+                        <StatusIndicator status={(product.stock > 0) ? 'default' : 'error'} />
+                    </div>,
+                    <div className="admin-products__mobile-item-container">
+                        <div className="admin-products__mobile-title">
+                            <Text size="medium" weight="bold">
+                                <FormattedMessage id="imagesHeading" />
+                            </Text>
+                        </div>
+                        <StatusIndicator status={(product.images.length > 0) ? 'default' : 'error'} />
+                    </div>,
+                    <div>
+                        <div className="admin-products__mobile-title">
+                            <Text size="medium" weight="bold">
+                                <FormattedMessage id="sectionsHeading" />
+                            </Text>
+                        </div>
+                        <Text size="medium">{getProductSections(product)}</Text>
+                    </div>,
+                    <div className="admin-products__mobile-item-container">
+                        <div className="admin-products__mobile-title">
+                            <Text size="medium" weight="bold">
+                                <FormattedMessage id="enabledHeading" />
+                            </Text>
+                        </div>
+                        <StatusIndicator status={(product.enabled === true) ? 'success' : 'default'} />
+                    </div>,
                 ]
             };
         });
diff --git a/src/components/pages/Admin/Products/AdminProducts.scss b/src/components/pages/Admin/Products/AdminProducts.scss
index d21c739d..4ac12155 100644
--- a/src/components/pages/Admin/Products/AdminProducts.scss
+++ b/src/components/pages/Admin/Products/AdminProducts.scss
@@ -6,6 +6,9 @@
 
 .admin-products__header {
     display: flex;
+    @media only screen and (max-width: 400px) {
+        flex-direction: column;
+    }
 }
 
 .admin-products__title {
@@ -46,6 +49,21 @@
     margin-top: 30px;
 }
 
+.admin-products__mobile-item-container {
+    @media only screen and (max-width: 400px) {
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+    }
+}
+
+.admin-products__mobile-title {
+    display: none;
+    @media only screen and (max-width: 400px) {
+        display: flex;
+        padding-right: 10px;
+    }
+}
 .admin-products__link {
     & > a {
         color: $color-primary;
diff --git a/src/components/pages/Admin/Products/AdminProductsAddForm.js b/src/components/pages/Admin/Products/AdminProductsAddForm.js
index b7302eab..3a5bed05 100644
--- a/src/components/pages/Admin/Products/AdminProductsAddForm.js
+++ b/src/components/pages/Admin/Products/AdminProductsAddForm.js
@@ -43,7 +43,7 @@ class AdminProductsAddForm extends React.Component {
             name[locale] = '';
         });
         
-        this.setState({name, sku: this.props.lastSKU.toString()});
+        this.setState({name, sku: Number(this.props.lastSKU) + 1});
         // Component styles
         require('./AdminProductsAddForm.scss');
     }
@@ -78,7 +78,7 @@ class AdminProductsAddForm extends React.Component {
 
         if (Object.keys(fieldErrors).length === 0) {
             this.props.onSubmitClick({
-                sku: this.state.sku,
+                sku: this.state.sku.toString(),
                 name: this.state.name
             });
         }
diff --git a/src/components/pages/Admin/Products/AdminProductsEdit.js b/src/components/pages/Admin/Products/AdminProductsEdit.js
index 017fee6c..335185c2 100644
--- a/src/components/pages/Admin/Products/AdminProductsEdit.js
+++ b/src/components/pages/Admin/Products/AdminProductsEdit.js
@@ -15,6 +15,11 @@ import ProductDetailsStore from '../../../../stores/Products/ProductDetailsStore
 import fetchProductAndCheckIfFound from '../../../../actions/Products/fetchProductAndCheckIfFound';
 import updateProduct from '../../../../actions/Admin/updateProduct';
 
+// Delete
+import DeleteHandler from '../../../common/deleteHandler/DeleteHandler';
+import deleteProduct from '../../../../actions/Admin/deleteProduct';
+import Modal from '../../../common/modals/Modal';
+
 // Required components
 import Button from '../../../common/buttons/Button';
 import Checkbox from '../../../common/forms/Checkbox';
@@ -49,7 +54,7 @@ class AdminProductsEdit extends React.Component {
         availableLocales: this.context.getStore(IntlStore).getAvailableLocales(),
         categories: this.context.getStore(CollectionsStore).getCollections(['category']),
         collections: this.context.getStore(CollectionsStore).getCollections(['collection']),
-        fieldErrors: {}
+        fieldErrors: {},
     };
 
     //*** Component Lifecycle ***//
@@ -97,6 +102,14 @@ class AdminProductsEdit extends React.Component {
         });
     }
 
+    handleDeleteProductClick = () => {
+        let product = this.state.product;
+        this.context.executeAction(deleteProduct, product.id);
+        let intl = this.context.intl;
+        let locale = intl.locale;
+        this.props.history.push(`/${locale}/adm/products`);
+    };
+
     //*** View Controllers ***//
 
     handleEnabledChange = () => {
@@ -290,6 +303,11 @@ class AdminProductsEdit extends React.Component {
                                     <FormattedMessage id="saveButton" />
                                 </Button>
                             </div>
+                            <div className="admin-products-edit__toolbar-item">
+                                <Button type="primary" onClick={this.handleDeleteProductClick} disabled={this.state.loading}>
+                                    <FormattedMessage id="deleteProduct" />
+                                </Button>
+                            </div>
                         </div>
                         :
                         null
diff --git a/src/constants/products.js b/src/constants/products.js
index 53a8bcdf..f8fbe7b3 100644
--- a/src/constants/products.js
+++ b/src/constants/products.js
@@ -12,6 +12,11 @@ export default keymirror({
     PRODUCTS_FIND_SUCCESS: null,
     PRODUCTS_FIND_ERROR: null,
 
+    // Delete item
+    PRODUCTS_ITEM_DELETE: null,
+    PRODUCTS_ITEM_DELETE_SUCCESS: null,
+    PRODUCTS_ITEM_DELETE_ERROR: null,
+
     // Products featured in homepage
     PRODUCTS_HOMEPAGE: null,
     PRODUCTS_HOMEPAGE_SUCCESS: null,
diff --git a/static/localizations/en.json b/static/localizations/en.json
index b4fea312..32a3fcb2 100644
--- a/static/localizations/en.json
+++ b/static/localizations/en.json
@@ -205,6 +205,8 @@
     "resetSuccessModalTitle": "Reset Requested",
     "retailPrice": "Retail Price",
     "saveButton": "Save",
+    "deleteProduct": "Delete",
+    "deleteProductModalTitle": "Confirm the deletion?",
     "savedAddresses": "Choose one of your saved addresses",
     "sections": "Featured in Sections",
     "sectionsHeading": "Sections",
diff --git a/static/localizations/es.json b/static/localizations/es.json
index 848226c8..56af80bb 100644
--- a/static/localizations/es.json
+++ b/static/localizations/es.json
@@ -205,6 +205,8 @@
   "resetSuccessModalTitle": "Pedido confirmado",
   "retailPrice": "Precio al por menor",
   "saveButton": "Guardar",
+  "deleteProduct": "Eliminar",
+  "deleteProductModalTitle": "Confirma la eliminacion?",
   "savedAddresses": "Elegí una de las direcciones guardadas",
   "sections": "Secciones Destacadas",
   "sectionsHeading": "Secciones",
diff --git a/static/localizations/ru.json b/static/localizations/ru.json
index ad90299c..01695412 100644
--- a/static/localizations/ru.json
+++ b/static/localizations/ru.json
@@ -212,6 +212,8 @@
     "resetSuccessModalTitle": "Пароль сброшен",
     "retailPrice": "Розничная цена",
     "saveButton": "Сохранить",
+    "deleteProduct": "Eliminar",
+    "deleteProductModalTitle": "Confirma la eliminacion?",
     "savedAddresses": "Выберите один из сохраненных адресов",
     "sections": "Показывать в секциях",
     "sectionsHeading": "Секции",
diff --git a/static/localizations/uk.json b/static/localizations/uk.json
index b637be97..57d3b1c7 100644
--- a/static/localizations/uk.json
+++ b/static/localizations/uk.json
@@ -205,6 +205,8 @@
     "resetSuccessModalTitle": "Запит на скидання створений",
     "retailPrice": "Роздрібна ціна",
     "saveButton": "Зберегти",
+    "deleteProduct": "Eliminar",
+    "deleteProductModalTitle": "Confirma la eliminacion?",
     "savedAddresses": "Оберіть одну із збережених адрес",
     "sections": "Показувати в секціях",
     "sectionsHeading": "Секції",
